ocs_ci/ocs/ui/helpers_ui.py

```diff
@@ -95,6 +98,157 @@ def format_locator(locator, string_to_insert):
     return locator[0].format(string_to_insert), locator[1]
 
 
+def create_storage_class_ui(
+    setup_ui,
+    encryption=False,
+    backend_path=None,
+    vault_namespace=None,
+    reclaim_policy="Delete",
+    provisioner="rbd",
+    vol_binding_mode="Immediate",
+    service_name=None,
+    kms_address=None,
+    tls_server_name=None,
+):
+    """
+    Test for creation of storage class with or without encryption via UI
+
+    Args:
+            setup_ui: login function on conftest file
+            encryption (bool): By default set to False, should be True for Encryption enabled Storage Class
+            backend_path (str): name of the vault backend path
+            vault_namespace (str): name of the vault namespace
+            reclaim_policy (str): value of the reclaim policy, it could be 'Delete' or 'Retain'
+            provisioner (str): type of provisioner used, it could be 'rbd' or 'cephfs'
+            vol_binding_mode (str): value of the volume binding mode, it could be 'WaitForFirstConsumer' or 'Immediate'
+            service_name (str): the default value is None which can be changed in the function call
+            kms_address (str): the default value is None which can be changed in the function call
+            tls_server_name (str): the default value is None which can be changed in the function call
+
+    Returns:
+            sc_name (str) if the storage class creation is successful, returns False otherwise
+    """
+    base_ui_obj = PageNavigator(setup_ui)
+
+    ocp_version = get_ocp_version()
+    pvc_loc = locators[ocp_version]["storage_class"]
+
+    base_ui_obj.navigate_storageclasses_page()
+    logger.info("Create Storage Class")
+    base_ui_obj.do_click(pvc_loc["create-sc"])
+    logger.info("Storage Class Name")
+    sc_name = create_unique_resource_name(
+        resource_description="test", resource_type="storageclass"
+    )
+    base_ui_obj.do_send_keys(pvc_loc["sc-name"], f"{sc_name}")
+    logger.info("Storage Class Description")
+    base_ui_obj.do_send_keys(pvc_loc["sc-description"], "this is a test storage class")
+    logger.info("Storage Class Reclaim Policy")
+    base_ui_obj.do_click(pvc_loc["reclaim-policy"])
+    if reclaim_policy == "Delete":
+        base_ui_obj.do_click(pvc_loc["reclaim-policy-delete"])
+    elif reclaim_policy == "Retain":
+        base_ui_obj.do_click(pvc_loc["reclaim-policy-retain"])
+
+    if ocp_version >= "4.9":
```

Comment:
Would it be possible to be compare the version as mentioned here: https://github.com/red-hat-storage/ocs-ci/pull/4872 ?
This would do away with any irregularities there might be when the version is 4.10 or higher.

Reply:
Ack